README.md

```diff
@@ -58,9 +65,42 @@ Then, [build an APK](https://developer.android.com/studio/build/building-cmdline
 deltachat@6012dcb974fe:/home/app$ ./gradlew assembleDebug
 ```
 
-If you don't want to use Docker or Podman, proceed to the next section.
+## Troubleshooting
+
+- `./gradlew assembleDebug` fails with `The SDK directory '/home/user/Android/Sdk' does not exist.`:
+  
+  You have to
+  - either: remove the line starting with `sdk.dir=` from your `local.properties` file
+  - or: execute `./gradlew assembleDebug` from outside the container
```

Comment:
Not sure this is a good idea. Core is linked using a toolchain from android NDK, so if you have a different version installed outside the container, this may cause incompatibility issues.

Probably better just say that this can be caused by `sdk.dir` in `local.properties` file created by Android Studio and let the user decide what to do about it.